std/common/doc.go

```diff
@@ -74,16 +77,31 @@ func Services(d core.DIDDoc) []did.Service {
 	}
 }
 
+func getEndpointType(t interface{}) string {
+	if epType, ok := t.(endpoint.EndpointType); ok {
+		switch epType {
+		case endpoint.DIDCommV1:
+			return "did-communication"
+		case endpoint.DIDCommV2:
+			return "DIDCommMessaging"
+		}
+	}
+	return ""
+}
+
 func SetServices(d core.DIDDoc, s []did.Service) {
+	defer err2.Catch(err2.Err(func(err error) {
```

Comment:
**Just** FYI: with the current err2 allows this catch-block to be shortened, and `cmd/root.go` hast `glog.CopyStandardLogTo("ERROR")` which gives pretty much the same result:
```go
defer err2.Catch()
```
Why this is a little better, it's the refactoring and copy-pasting, not the amount of lines some much.